style.css

```diff
@@ -925,15 +944,241 @@ textarea {
     display: none;
   }
 
-  .top-nav {
+  .toolbar-ul {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 40vh;
+    font-family: "Poppins", sans-serif;
+    list-style: none;
+    text-decoration: none;
+  }
+
+  /**  Modal styling  **/
+  .popup-window {
+    left: 10rem;
+    top: 77px;
+    display: none;
+    justify-content: center;
+    align-items: center;
+    width: 1156px;
+    position: fixed;
+    z-index: 2;
+    background: white;
+    overflow-y: scroll;
+  }
+
+  .popups-container {
     display: flex;
     flex-direction: row;
+    width: 85%;
+    border-radius: 10px;
+    background-color: white;
+    margin: 30px 0 30px 0;
+  }
+
+  .popup-img {
+    position: absolute;
+    width: 1080px;
+    height: 586px;
+    left: 0;
+    top: 0;
+    margin-top: 7rem;
+    margin-left: 30px;
+  }
+
+  .top-block {
+    display: flex;
+    flex-direction: column;
     align-items: center;
+    gap: 12px;
+    position: absolute;
+    width: 97%;
+    justify-content: space-between;
+    height: 84px;
+    left: 24px;
+    top: 24px;
+    background-color: #fff;
+  }
+
+  .frame1 {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
     padding: 0;
-    gap: 8px;
-    width: 259px;
+    gap: 12px;
+    width: auto;
+    margin-right: 5rem;
+  }
+
+  .frame2 {
+    display: flex;
+    flex-direction: row;
+    padding: 0;
+    gap: 12px;
+    margin-top: 2.5rem;
+    width: auto;
+    margin-right: 2.5rem;
+  }
+
+  .closeIcon-popup {
+    color: #67798e;
+    display: flex;
+    margin: auto;
+    margin-left: 60rem;
+    font-size: 30px;
+    cursor: pointer;
+  }
+
+  .popups-proj-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    color: #091e42;
+    width: auto;
+    height: 40px;
+    padding-left: 5rem;
+  }
+
+  .right-block {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    position: absolute;
+    width: auto;
+    height: 157px;
+    right: 24px;
+    top: 781px;
+  }
+
+  .top-block ul {
+    padding: 0;
+    gap: 12px;
+    width: 1056px;
+    height: 36px;
+    margin-top: -50px;
+  }
+
+  .popups-project-p-desktop {
+    display: flex;
+  }
+
+  .divider {
+    display: none;
+    height: 0;
+    left: 4px;
+    right: 4px;
+    color: #ebecf0;
+    margin: auto;
+    overflow-y: scroll;
+  }
+
+  .divider2 {
+    display: flex;
+    margin: auto;
+    margin-top: 5rem;
+  }
+
+  .overlay {
+    height: 100%;
+    width: 100%;
+    position: absolute;
+    bottom: -260px;
+    background: #c1c7d0;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(3px);
+    z-index: 1;
+  }
```

Comment:
- [ ] **[Optional]** I think it would be better to change the ```position: absolute;``` to ```position: fixed;``` to fix the overlay to you page to hide the web contents while scrolling the page for the best user experience 👍 👍  

You may follow this[ link](https://developer.mozilla.org/en-US/docs/Web/CSS/position) for more info on CSS positions